standard/flags.lua

```diff
@@ -95,9 +110,9 @@ function Flags.CountryCode(flagName, format)
 
 	if flagKey then
 		if format == 'alpha3' then
-			return Flags._getAlpha3CodesByKey()[flagKey]
+			return Flags._getAlpha3CodesByKey()[flagKey] or Flags._getLanguageCodesByKey()[flagKey]
 		else
-			return Flags._getAlpha2CodesByKey()[flagKey]
+			return Flags._getAlpha2CodesByKey()[flagKey] or Flags._getLanguageCodesByKey()[flagKey]
```

Comment:
Only the alpha2 handling is correct, requesting alpha3 format with a language will return an alpha2?